lib/authentication/view/edit_profile_page.dart

```diff
@@ -48,8 +49,16 @@ class _EditProfilePageState extends State<EditProfilePage> {
     super.initState();
     final authProvider = Provider.of<AuthProvider>(context, listen: false);
     authProvider.isVerified.then((value) => setState(() => isVerified = value));
-    authProvider.getProfilePictureURL().then((value) =>
-        setState(() => {if (value != null) imageWidget = NetworkImage(value)}));
+    authProvider.getProfilePictureURL().then((value) => setState(() => {
+          imageWidget = value != null
+              ? NetworkImage(value)
+              : const AssetImage('assets/illustrations/undraw_profile_pic.png'),
+        }));
+    uploadButtonController = UploadButton(
+            pageType: true,
+            controller:
+                UploadButtonController(onUpdate: () => setState(() => {})))
+        .controller;
```

Comment:
Uhm, just initialize the UploadButtonController. The UploadButton is pointless here.